Templates/MonoGame.Templates.CSharp/content/MonoGame.2D.StartKit/___SafeGameName___.Android/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="___SafeGameName___.Android" android:versionCode="1" android:versionName="1.0">
+	<uses-sdk android:minSdkVersion="23" android:targetSdkVersion="34" />
```

Comment:
you can remove the `uses-sdk` element completely. This is filled in by the `SupportedOSPlatformVersion` property in the csproj.

Reply:
Thanks, fixed.